pkg/invoke/os_linux.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2016 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
```

Comment:
I'd prefer that we name this file `os_unix.go` and expand the list of allowed OSs.  

That way, the package may be built on Mac and other systems.  E.g.


```go
// +build darwin dragonfly freebsd linux netbsd openbsd solaris
```

For an example, see [`golang.org/x/sys/unix/env_unix.go`](https://github.com/containernetworking/cni/blob/master/vendor/golang.org/x/sys/unix/env_unix.go#L5)